src/views/Editor.vue

```diff
@@ -91,29 +91,21 @@
 		<b-alert variant="danger" :show="!!error" dismissible @dismissed="error=null"><i class="fas fa-ban"></i> API error: {{ error }}</b-alert>
 		<b-alert variant="warning"><font-awesome-icon icon="info"></font-awesome-icon> Publishing: I understand that publishing this dataset:</b-alert>
 
-		<b-card id="publish-verification-card" variant="dark" bg-variant="dark" text-variant="white" v-if="showPublishConfirmation">
-			<h3 slot="title">
-				<font-awesome-icon icon="info" fixed-width />
-				Publishing
-			</h3>
-			<p class="card-text">I understand that publishing this dataset...</p>
-				<ul class="list-unstyled">
-					<li class="font-italic">... will make it available publicly</li>
-					<li class="font-italic">... marks it as ready and enables editing restrictions</li>
-				</ul>
-			<p></p>
-			<div class="float-right">
-				<b-button id="publish-verification-card-button-cancel" variant="outline-light" class="ml-3" @click="showPublishConfirmation = false"><font-awesome-icon icon="times" fixed-width /> cancel</b-button>
-				<b-button id="publish-verification-card-button-help" variant="danger" class="ml-3" @click="showPublishConfirmation = false" v-if="false"><font-awesome-icon icon="info" fixed-width /> help</b-button>
-				<b-button id="publish-verification-card-button-publish" variant="success" :disabled="saving" class="ml-3" @click="publish()"><font-awesome-icon icon="cloud-upload-alt" fixed-width /> publish</b-button>
-			</div>
-		</b-card>
-
 
 		<!-- Modals -->
 		<dataset-json-modal id="dataset-json-modal"></dataset-json-modal>
 		<dataset-overview-modal id="dataset-overview-modal"></dataset-overview-modal>
-		<publish-modal id="publish-modal" :error="publishError" @hidden="publishError = null"></publish-modal>
+		<b-modal ref="publishModal" id="publishModal" title="Publish dataset?"
+			ok-title="Publish" cancel-variant="primary" ok-variant="success" @ok="publish">
+			<p :style="{'text-align': 'center'}">
+				<p>I understand that publishing this dataset...</p>
+				<ul class="list-unstyled">
+					<li class="font-italic">... will make it available publicly</li>
+					<li class="font-italic">... marks it as ready and enables editing restrictions</li>
```

Comment:
Do we want to keep the leading ... before the actual text?

Reply:
I will change it